Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -318,7 +318,7 @@ private void InitializeServiceClient(TelemetryConfiguration configuration)
         private static string GetInstanceName(TelemetryConfiguration configuration)
         {
             // we need to initialize an item to get instance information
-            var fakeItem = new MetricTelemetry();
```

Comment:
We’re definitely safe for RoleInstance attribute. Initializers can, naturally, check the type of the item but Azure one does not